src/sentry/tsdb/base.py

```diff
@@ -370,10 +380,19 @@ def get_sums(self, model, keys, start, end, rollup=None, environment_id=None, us
             rollup,
             environment_ids=[environment_id] if environment_id is not None else None,
             use_cache=use_cache,
+            jitter_value=jitter_value,
         )
         sum_set = {key: sum(p for _, p in points) for (key, points) in range_set.items()}
         return sum_set
 
+    def _add_jitter_to_series(self, series, start, rollup, jitter_value):
+        if jitter_value:
+            jitter = jitter_value % rollup
+            if (start - to_datetime(series[0])).total_seconds() < jitter:
```

Comment:
Could there be scenarios where `series` is empty?

Reply:
In practice no, but it's probably worth guarding against it in case this is used elsewhere in the future.